pycolmap/ci/install-centos.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash
```

Comment:
nit: these scripts only install colmap but not pycolmap. Maybe we call the scripts install-colmap-centos.sh etc.?